### ISSUE TITLE
auditwheel depends on a specific patch version of wheel

### ISSUE THREAD

**Issue by mdboom** [pypa/auditwheel#274]
auditwheel currently depends on a [specific patch version of wheel](https://github.com/pypa/auditwheel/blob/master/requirements.txt#L1).  Is this intentional?  I would have assumed it would ignore the patch version.  This currently causes problems for projects running dependabot, since `wheel` is held back: https://github.com/mozilla/glean/pull/1387

**Comment by mayeut** [comment]
@mdboom,

It's intentional. `wheel` has no public API yet and `auditwheel` is using a private API of `wheel` which is perfectly free to change between patch versions as they're just implementation details in `wheel` (and they did in the past causing all sort of issues in `auditwheel`).
At one time I opened #191 which has been closed when `auditwheel` pinned `wheel` version.
I was thinking of reopening the issue (and do the related work) to vendor `wheel` until it gets a public API (which is work in progress). I'm also getting annoyed by this problem when `dependabot` tries to update wheel (as it should).
